aidial_adapter_vertexai/universal_api/token_usage.py

```diff
@@ -19,19 +9,10 @@ class TokenUsage(BaseModel):
     def total_tokens(self) -> int:
         return self.prompt_tokens + self.completion_tokens
 
-    def __add__(self, other: "TokenUsage") -> "TokenUsage":
-        return TokenUsage(
-            prompt_tokens=self.prompt_tokens + other.prompt_tokens,
-            completion_tokens=self.completion_tokens + other.completion_tokens,
-        )
+    def accumulate(self, other: "TokenUsage") -> "TokenUsage":
+        self.prompt_tokens += other.prompt_tokens
+        self.completion_tokens += other.completion_tokens
+        return self
 
-    def to_dict(self) -> TokenUsageDict:
-        ret: TokenUsageDict = {
-            "prompt_tokens": self.prompt_tokens,
-            "total_tokens": self.total_tokens,
-        }
-
-        if self.completion_tokens != 0:
-            ret["completion_tokens"] = self.completion_tokens
-
-        return ret
+    def __add__(self, other: "TokenUsage") -> "TokenUsage":
```

Comment:
Is this method really needed? It looks like the way it is used now can be replaced with the accumulate method.

Reply:
Fixed. Removed.